### PR TITLE
adding missing EasyMock.verify calls

### LINKED ISSUE #1408: Missing EasyMock.verify in unit tests
In some unit tests, before resetting mocked objects or after mocked objects get used, `EasyMock.verify` does not get invoked on mocked objects. So, we should add it.

For example, in `UserTaskManagerTest.java` and `KafkaCruiseControlServletEndpointTest.java`. There could be more cases. So, I suggest thoroughly iterate all cases where `EasyMock` gets used.

### PR DESCRIPTION
**Pull request by jacobhell**
This PR resolves #1408

### CODE REVIEW

File: cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java [hunk 1]
```diff
@@ -53,6 +53,7 @@ public void testCreateUserTask() {
     Assert.assertEquals(userTaskHeader.getValue(), UserTaskManager.USER_TASK_HEADER_NAME);
     Assert.assertEquals(userTaskHeaderValue.getValue(), testUserTaskId.toString());
     Assert.assertEquals(future, future1);
+    EasyMock.verify(mockHttpServletResponse);
```
Comment (efeg) [inline cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java:56]: Should this be `EasyMock.verify(mockHttpServletResponse, mockHttpSession, mockUUIDGenerator);`?

**Review by efeg — CHANGES_REQUESTED** [review]
I see missing cases in both the modified and unmodified tests -- i.e. in some cases `verify` does not include all mocks (left an example instance in review), in others `verify` is completely missing (e.g. verify for `HttpServletRequest` mock in `UserTaskManagerTest#prepareRequest(HttpSession, String, String, Map<String, String []>)`).

Can we add the missing EasyMock.verify in relevant unit tests?

**Comment by efeg** [comment]
Closing this PR due to inactivity.

**Comment by shreeraam-ramachandran** [comment]
Hi can i work on this issue

**Pull request closed without merging**